dataprofiler/data_readers/csv_data.py

```diff
@@ -65,7 +65,7 @@ def __init__(self, input_file_path=None, data=None, options=None):
         self._selected_data_format = options.get("data_format", "dataframe")
         self._delimiter = options.get("delimiter", None)
         self._selected_columns = options.get("selected_columns", list())
-        self._header = options.get("header", 0)  # 0 for pandas to auto get
+        self._header = options.get("header", 'auto')
```

Comment:
This needs to be documented somewhere in the API. It should also be added to the `CSVData class` README section

Reply:
done